examples/takeoff_and_land.py

```diff
@@ -0,0 +1,23 @@
+import asyncio
+
+from dronecode_sdk import (AsyncPluginManager,
+                           Action)
+
+manager = AsyncPluginManager(host="127.0.0.1")
+action = Action(manager)
+
+async def run():
+
+    arm_result = await action.arm()
+    print("Arm result: " + str(arm_result[1].action_result.result_str))
```

Comment:
No need to call `str` afaik

Reply:
Right, it was `str(arm_result[1])` before, and I did not realize that. Thanks!